modules/33acrossIdSystem.js

```diff
@@ -73,14 +73,41 @@ function calculateQueryStringParams(pid, gdprConsentData) {
     params.gdpr_consent = gdprConsentData.consentString;
   }
 
-  const fp = storage.getDataFromLocalStorage(STORAGE_FPID_KEY);
+  const fp = getStoredValue(STORAGE_FPID_KEY, storageConfig);
   if (fp) {
     params.fp = fp;
   }
 
   return params;
 }
 
+function handleFpId(fpId, storageConfig = {}) {
+  if (!fpId) {
+    return storage.removeDataFromLocalStorage(STORAGE_FPID_KEY);
+  }
+
+  storeValue(STORAGE_FPID_KEY, fpId, storageConfig);
+}
+
+function storeValue(key, value, storageConfig = {}) {
+  if (storageConfig.type === STORAGE_TYPE_COOKIES && storage.cookiesAreEnabled()) {
+    const expirationInMs = 60 * 60 * 24 * 1000 * storageConfig.expires;
+    const expirationTime = new Date(Date.now() + expirationInMs);
+
+    storage.setCookie(key, value, expirationTime.toUTCString(), 'Lax');
```

Comment:
that's correct, the subdomain is included. I've verified this behavior and that's how most of the IdSystems set the cookies, without a hostname, since that part is handled by the storage manager util.